### ISSUE TITLE
feat(linter): --rules should only show registered

### ISSUE THREAD

**Issue by Yexiaoxing** [oxc-project/oxc#1751]
Version: 0.0.21

Repro steps:

```bash
npx oxlint@latest --rules
```

Current behaviour:
1. It shows the registered rules.
2. It also runs the check.

Expected behaviour:
1. `--rules` should only show rules but not run it. Or provide a way to do so.

**Comment by camc314** [comment]
Should be fixed by https://github.com/oxc-project/oxc/pull/1747

**Comment by camc314** [comment]
@Yexiaoxing going to close this - will be fixed in the next release - can you reopen if you still see an issue please? 

thanks!
